examples/kubernetes/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
I think here you should set up the stack too?

Reply:
Yeah we'll need the whole step setup here as well - check the kubeflow example

Reply:
Removed setup.sh.